float64 value1
float64 value2
---
bool result
